docs/mnist_example/README.md

```diff
@@ -196,19 +201,15 @@ int main() {
   int64_t rank = 4;
   int64_t shape[] = {1, 1, 28, 28};
 
-  // Create a tensor using omTensorCreateWithOwnership (returns a pointer to OMTensor).
+  // Create a tensor using omTensorCreateWithOwnership (returns a pointer to the OMTensor).
   // When the parameter, owning is set to "true", the OMTensor will free the data
-  // pointer (img_data) upon destruction. If owning is set to false, the data pointer will not be
-  // freed upon destruction. 
-  OMTensor *tensor = omTensorCreateWithOwnership(img_data, shape, rank, ONNX_TYPE_FLOAT, /*owning=*/true);
-
-  // Create a tensor list using omTensorListCreateWithOwnership (returns a pointer to the OMTensorList).
-  // When the parameter, owning is set to "true", the OMTensorList will free the tensor array
-  // upon destruction. If owning is set to false, the tensor array will not be
-  // freed upon destruction. It it important to note, that no matter the value of the 
-  // ownership flag, every tensor will be destroyed.
+  // pointer (img_data) upon destruction. If owning is set to false, the data pointer will
+  // not be freed upon destruction.
+  OMTensor *tensor = omTensorCreate(img_data, shape, rank, ONNX_TYPE_FLOAT, /*owning=*/true);
```

Comment:
We are only changing TensorList....Let me check the file again to see what happened.

Reply:
Okay this is how it reads:

```
 // Create a tensor using omTensorCreateWithOwnership (returns a pointer to the OMTensor).
  // When the parameter, owning is set to "true", the OMTensor will free the data
  // pointer (img_data) upon destruction. If owning is set to false, the data pointer will
  // not be freed upon destruction.
  OMTensor *tensor = omTensorCreate(img_data, shape, rank, ONNX_TYPE_FLOAT, /*owning=*/true);

  // Create a tensor list using omTensorListCreate (returns a pointer to the OMTensorList).
  inputTensors[0] = tensor;
  OMTensorList *tensorListIn = omTensorListCreate(inputTensors, inputNum);

  // Compute outputs.
  OMTensorList *tensorListOut = run_main_graph(tensorListIn);

  // Extract the output. The model defines one output of type tensor<1x10xf32>.
  OMTensor *y = omTensorListGetOmtByIndex(tensorListOut, 0);
  float *prediction = (float *)omTensorGetDataPtr(y);
```

Reply:
Because some of the wording was redundant with `omTensorListCreateWithOwnership`, once I removed it....it now looks a bit weird but reads nicely if you view the file from my branch (without the red and green highlighted changes).

Reply:
But doesn't this change `OMTensor *tensor = omTensorCreateWithOwnership(...)` to `OMTensor *tensor = omTensorCreate(...)`?  When I look at the definition of [omTensorCreate()](https://github.com/onnx/onnx-mlir/blob/main/src/Runtime/OMTensor.inc#L193), I don't see an `owning` parameter as an option. From what I can see, that method [hardcodes a false](https://github.com/onnx/onnx-mlir/blob/main/src/Runtime/OMTensor.inc#L205).

Reply:
I see the confusion...give me one second.

Reply:
I updated this